modules/login/templates/form_passwordexpiry.tpl

```diff
@@ -1,3 +1,7 @@
+
+<script type="text/javascript" src="{$baseurl}/js/passwordVisibility.js"></script>
+<link rel="stylesheet" href="/css/password.css" type="text/css" />
```

Comment:
Is there a reason these can't be added to getJSDependencies and getCSSDependencies of the password expiry page's class? (And similarly, the script tag added to a `.js` file for the module..)

Reply:
@driusan Fixed!